app/views/moderations/mod.html.erb

```diff
@@ -72,10 +72,10 @@
   <% if current_user.has_role?(:super_admin) && @moderatable.class.name == "Article" %>
     <h3> <a href="<%= @moderatable.path %>/edit">Edit Post</a> |
       <a href="/resource_admin/articles/<%= @moderatable.id %>" data-no-instant>ResourceAdmin:Article</a> |
-      <a href="/admin/articles/<%= @moderatable.id %>" data-no-instant>Admin:Article</a></h3>
+      <a href="<%= admin_article_path(@moderatable.id) %>" data-no-instant>Admin:Article</a></h3>
   <% elsif current_user.has_role?(:super_admin) && @moderatable.class.name == "Comment" %>
     <h3> <a href="/admin/comments/<%= @moderatable.id %>" data-no-instant>Admin:Comment</a> |
```

Comment:
I'm not sure about this route: We don't seem to have an admin_comment_path(id) helper because the routes only have an index route defined `  resources :comments, only: [:index]`

Reply:
Is there a way to trigger that if manually in the UI? It looks like it's currently broken

Reply:
I did manage to trigger it and this is broken because this path does not exist. See screenshots below. I'm going to add an issue to fix this separately.  

<img width="1186" alt="Screenshot 2021-04-26 at 14 34 47" src="https://user-images.githubusercontent.com/2786819/116084037-a39d3f80-a69d-11eb-8a97-6f74ae4de0a7.png">

<img width="1184" alt="Screenshot 2021-04-26 at 14 34 56" src="https://user-images.githubusercontent.com/2786819/116084189-c9c2df80-a69d-11eb-865e-65741cb08d4c.png">